### ISSUE TITLE
Some feedback

### ISSUE THREAD

**Issue by HellsReindeer** [OrangeNote/TwitchVotesItems#1]
Mod worked for a while however it seems to break after a while, voting did not seem to register, current theory is that inputs that aren't options seem to break it but cannot guarantee this is the case.
Edit: After testing some more this doesn't seem to be the issue ^^

Some suggestions:
Create a config file capable of changing how long the poll lasts to adjust for stream delays.
Also in the config maybe setting a limiter on limiting how many voters?
Possibly make a command that refreshes the mod, and if possible can only be used by twitch mods?

**Comment by OrangeNote** [comment]
Thanks for the report, it never happened to me. Can you please be more precise about what wasn't working, just to be sure?

- The config file is planned and it will at least include the poll duration.
- How would the limter work and why would you want to introduce that? End the vote prematurely if there are enough votes?
- About refreshing the mod: not a high priority for now. Classes are a bit messy and I don't know if I can make it reload out of the box without issues. I'll see what I can do.
- Yeah currently the mod is active even if you're not connected to twitch, and it's kinda pointless, I'll disable that in the future.

**Comment by Bge508** [comment]
I can add a little bit to this.

- The mod crashes the game when teleporting to the 3rd stage
- The mod, after a certain point, no longer detects chat votes? I can't really tell what is going on, but even if people vote, it will always choose option 1.
- The 3 items that pop-up when opening a chest sometimes doesn't show anymore.

This was through about ~4 hours of testing and those were the biggest things I experienced that Hell's didn't hit on. The poll duration was super quick and people had no time to vote, but that one was discussed.

**Comment by OrangeNote** [comment]
About the third stage, it might be the timed chest. I tested it by skipping stages with cheats and it seemed to work. I'll see what happens in a normal run and when the timer expires.

About the rest, it's hard to tell if it's not reproducible. I'll keep working on it (right now it's in a very rough state) and I'll test more. Thanks!

**Comment by OrangeNote** [comment]
Quick update: I added config files for some options. Please test again while keeping the chat disabled (`DisplayTwitchMessages = false` in the TwitchIntegration bepinex config).

**Comment by Zach0813** [comment]
> Mod worked for a while however it seems to break after a while, voting did not seem to register, current theory is that inputs that aren't options seem to break it but cannot guarantee this is the case.
> Edit: After testing some more this doesn't seem to be the issue ^^
> 
> Some suggestions:
> Create a config file capable of changing how long the poll lasts to adjust for stream delays.
> Also in the config maybe setting a limiter on limiting how many voters?
> Possibly make a command that refreshes the mod, and if possible can only be used by twitch mods?

I would agree on making the configuration file local so each user can edit their own timer length and even chat message layout.

**Comment by Zach0813** [comment]
Suggestion to add "Vote Message" and "Win Message" customization to the planned config file.
